2024/json-ld-wg/index.html

```diff
@@ -155,6 +155,9 @@ <h2>Scope</h2>
 					The Working Group will maintain the JSON-LD specifications (i.e., <a href="https://www.w3.org/TR/json-ld11/">JSON-LD 1.1</a>, <a href="https://www.w3.org/TR/json-ld11-api/">JSON-LD 1.1 API</a>, <a href="https://www.w3.org/TR/json-ld11-framing/">JSON-LD 1.1 Framing</a>) that together provide a JSON format for Linked Open Data to interoperate at web-scale, in a method which is familiar to and usable by web-focused software engineers.
 					<!-- Note that the JSON-LD APIs are not browser specific; while appropriate for use within browsers, they are not limited to such use, and there is no requirement for browsers to implement them natively. -->
 				</p>
+        <p>
+          It will also develop new Recomendation Track deliverables, based on work incubated by the <a href="https://www.w3.org/community/json-ld/">JSON for Linking Data Community Group</a>, specifying the use JSON-LD algorithms with similar formats (YAML, CBOR).
```

Comment:
```suggestion
          It will also develop new Recommendation Track deliverables, based on work incubated by the <a href="https://www.w3.org/community/json-ld/">JSON for Linking Data Community Group</a>,
          specifying the use of JSON-LD algorithms with similar formats (YAML, CBOR, and more).
```

Maybe we'll want to do TOML-LD,  at some point, why not?